jdaviz/configs/cubeviz/plugins/spectral_extraction/spectral_extraction.py

```diff
@@ -109,6 +109,8 @@ def __init__(self, *args, **kwargs):
             "please load data to enable this plugin."
         )
 
+        self._sync_feature_flag('ff_cone_apertures', 'cone_apertures')
```

Comment:
this synchronizes the boolean traitlet in the plugin to get updates when the feature flag is toggled